common/src/main/java/com/scalar/dl/ledger/error/CommonError.java

```diff
@@ -215,6 +215,12 @@ public enum CommonError implements ScalarDlError {
       ""),
   INVALID_NAMESPACE_NAME(
       StatusCode.INVALID_ARGUMENT, "019", "The namespace name is invalid. Name: %s", "", ""),
+  NAMESPACE_NOT_SUPPORTED_IN_DEPRECATED_CONTRACT(
+      StatusCode.INVALID_ARGUMENT,
+      "020",
+      "The namespace-aware interfaces are not supported in the deprecated contract.",
```

Comment:
Can we change this as follows? This makes the wording more generic, but please let me know if there's some nuance that I'm not understanding here:

```suggestion
      "Namespace-aware interfaces are not supported in deprecated contracts.",
```